docs/release_notes/v0.6.0.md

```diff
@@ -0,0 +1,250 @@
+
+# Dapr 0.6.0
+ 
+We're happy to announce the release of Dapr 0.6.0! 
+
+**Highlights**
+
+* Introduced *new* liveness/readiness probe endpoint for Dapr system/sidecar as well as actor service health API in User application
```

Comment:
The endpoint is for Dapr Side car container health only.

Reply:
fixed.